pkg/operator/types.go

```diff
@@ -53,3 +53,31 @@ func MakeHostAliases(input []monitoringv1.HostAlias) []v1.HostAlias {
 
 	return output
 }
+
+const ShardLabel = "operator.prometheus.io/shard"
+const ResourceNameLabel = "operator.prometheus.io/name"
+
+func MakeK8sTopologySpreadConstraint(podLabels map[string]string, tsc monitoringv1.TopologySpreadConstraint) v1.TopologySpreadConstraint {
```

Comment:
Don't we need to match also on the `app.kubernetes.io/name` label to differentiate between server and agent pods?

Reply:
Updated!
Can you check it?